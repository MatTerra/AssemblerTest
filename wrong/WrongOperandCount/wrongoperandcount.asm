section data
space 1
const
section text
start: mul start,start
add
stop
