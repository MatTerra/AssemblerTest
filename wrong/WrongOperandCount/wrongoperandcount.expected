Line 2 Invalid operand count for space - Syntax
Line 3 Invalid operand count for const - Syntax
Line 5 Invalid operand count for mul - Syntax
Line 5 Invalid operand count for add - Syntax
